common/persistence/visibility/store/elasticsearch/processor_test.go

```diff
@@ -128,6 +130,13 @@ func (s *processorSuite) TestNewESProcessorAndStartStop() {
 	p.Stop()
 	s.Nil(p.mapToAckFuture)
 	s.Nil(p.bulkProcessor)
+
+	// Confirm processor does not panic on new requests after stopping
+	request := &client.BulkableRequest{}
+	visibilityTaskKey := "test-key"
+	future1 := p.Add(request, visibilityTaskKey)
+	_, err := future1.Get(context.Background())
+	s.ErrorContains(err, "shut down")
```

Comment:
nit: this seems easy to break. Probably define the shutdown error somewhere and check against the defined error.